tests/app.js

```diff
@@ -40,9 +40,9 @@ import * as remoteConfigModular from '@react-native-firebase/remote-config';
 import '@react-native-firebase/storage';
 import * as storageModular from '@react-native-firebase/storage';
 import jet from 'jet/platform/react-native';
-import React from 'react';
 import { AppRegistry, Button, NativeModules, Text, View } from 'react-native';
 import DeviceInfo from 'react-native-device-info';
+import React from 'react';
```

Comment:
Please avoid unnecessary changes in PRs, this file would not be touched at all except for this change, which is not related to the PR, so it just adds noise to the diff / increases review burden

Reply:
Done